### PR TITLE
Publish v0.1.24

### PR DESCRIPTION
**Pull request by raoulmillais**

### CODE REVIEW

File: bin/wasm-node/javascript/smoldot-demo-db.json [hunk 1]
```diff
@@ -0,0 +1 @@
...ey":"c1c8b7680700472f35c23d539c36c87895724189c2158be4c9d8c818bc0a3dd6","weight":1},{"public_key":"404b31a6663344c68c7e7c2a1cf4a767f53a525fe022f80da283d2c8bb13686d","weight":1},{"public_key":"39fd30cff65229eb5137e7020f5b167d56f4ddc7b94838578a3c2133bb06de3a","weight":1},{"public_key":"5928686fa1be237692bd9896180cc83ee9e58c7b54872446cf72ef18c4caae7e","weight":1},{"public_key":"ca6fda68419e374351c2fc8314b2b1b636932390a23224d23c30197a20ec4cb2","weight":1},{"public_key":"31dae797bbac0e27b901355cc90446369dcd15d8f965933b57723ee389670d55","weight":1},{"public_key":"169da96fe889fe19f2e9463c4cb730b33473a561f0a15d5581ca7c52362a252d","weight":1},{"public_key":"b6f9a5a4b86a88db316afb7e2959804fd6b1e3e5774035e4e0179dc7762b1b63","weight":1},{"public_key":"7cf015bc8c5d14382b396df342b9e14a07507af1f8c96e0532a1b93b70c7166c","weight":1},{"public_key":"98ab15a1b8374e8e6dc53f594d8a6c1c481c71fe1c32fe5b58708e78126c2229","weight":1},{"public_key":"d1226fadc01b5271da540ad505db93e2f855b2b6cc2500f9b3878e63660512d4","weight":1},{"public_key":"07cbf08a347b44d8c3e9e661c02f1a9d1def2a324bad2c50965f6e5907550f04","weight":1},{"public_key":"f79cf74c9dba0d4803234f266947bbc058a1c70541b3f7c3f5db2448d41bd4d2","weight":1}]}
```
Comment (tomaka) [inline bin/wasm-node/javascript/smoldot-demo-db.json:1]: Please remove

Comment (raoulmillais) [reply bin/wasm-node/javascript/smoldot-demo-db.json:1]: Damn it I thought I'd removed that from my commit.

Comment (raoulmillais) [reply bin/wasm-node/javascript/smoldot-demo-db.json:1]: Should i add it to .gitignore too?

**Review by tomaka — APPROVED** [review]

File: bin/wasm-node/javascript/smoldot-demo-db.json (hunk 1 above)
Comment (tomaka) [reply bin/wasm-node/javascript/smoldot-demo-db.json]: The database isn't working anymore. No code is reading/writing this file anywhere. So no

**Pull request merged**